<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationsLayerSettingsClass</class>
 <widget class="QWidget" name="NotificationsLayerSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotificationsLayerSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="popupBox">
     <property name="text">
      <string>Show popup windows</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notifyBox">
     <property name="title">
      <string>Notify when</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="signOnCheckBox">
        <property name="text">
         <string>Contact sign on</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="signOffCheckBox">
        <property name="text">
         <string>Contact sign off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="typingCheckBox">
        <property name="text">
         <string>Contact typing message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="changeStatusCheckBox">
        <property name="text">
         <string>Contact change status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="messageBox">
        <property name="text">
         <string>Message received</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
